src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -89,6 +89,7 @@
   <!-- .NET Standard 1.7 -->
   <ItemGroup Condition="'$(PackageTargetFramework)' == 'netstandard1.7' and '$(TargetGroup)' != 'net46' and '$(TargetGroup)' != 'net463'" >
     <Compile Include="System\IO\FileStreamBase.NetStandard17.cs" />
+    <Compile Include="System\IO\FileStream.NetStandard17.cs" />
```

Comment:
Why is this in a separate file? I would expect that we will just redist the pre-netstandard1.7 binaries; and just build the netstandard17 version from the sources.